resource-watch/resource-watch
# Explore Discover section

![imagen](https://user-images.githubusercontent.com/545342/76617774-4a209280-6527-11ea-9d69-ffb6243ebb18.png)

## Overview
This PR implements the `Discover` section.
⚠️ The data for the sub-section called _Related topics_ has been temporarily mocked until we have a way to dynamically load it. However, there's a `config` service that has implemented already for that purpose.

## Testing instructions
Go to http://localhost:9000/data/explore?section=Discover and verify that the data for all four sub-sections is working fine.
